skbeam/core/mask.py

```diff
@@ -108,3 +109,81 @@ def threshold_mask(images, threshold, mask=None):
         if len(bad_pixels[0]) != 0:
             mask[bad_pixels] = 0
         yield mask
+
+
+def mask_edge(img_shape, edge_size):
+    """
+    Mask the edge of an image
+
+    Parameters
+    -----------
+    img_shape: tuple
+        The shape of the image
+    edge_size: int
+        Number of pixels to mask from the edge
+    Returns
+    --------
+    1darray:
+        The raveled mask array, bad pixels are 0
+    """
+    mask = np.ones(img_shape)
+    mask[:, :edge_size] = 0
+    mask[:, -edge_size:] = 0
+    mask[:edge_size, :] = 0
+    mask[-edge_size:, :] = 0
+    return mask.ravel().astype(bool)
```

Comment:
I apologize if I gave you the wrong impression in a previous discussion. I think I am generally in favor of leaving masks in the same shape as the data set that they are supposed to be masking. 

Why are you in favor of having masks be 1D?